.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+## Proposed Changes
+
+- Change
+- Update
+- Fix
+
+## Relates To
+
+- Issue #0
+
+## Notify
+
+- @octocat
+- @beetboxvm/octoteam
```

Comment:
cool, thanks for the explanation
@thom8 what would be the correct groups for notification